utils/utils.js

```diff
@@ -26,11 +44,15 @@ errAndMsg = (channel, err) => {
     channel.send(`${errorPhrases[getRandomIndex(errorPhrases)]} ${err}`);
 };
 
+isEmptyObj = (obj) => {
+    return Object.keys(obj).length === 0 && obj.constructor === Object;
```

Comment:
How come it's necessary to check that it's an object? I think `Object.keys(obj).length === 0` is probably enough.